Berksfile

```diff
@@ -2,4 +2,8 @@ source 'https://supermarket.chef.io'
 
 cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
 
+# forked to add centos 8 support to upstream cookbook
+# https://github.com/atomic-penguin/cookbook-nfs/pull/114
+cookbook 'nfs', git: 'git@github.com:Vancelot11/cookbook-nfs', branch: 'vancelot/c8'
```

Comment:
Looks like this was merged and released: https://supermarket.chef.io/cookbooks/nfs#changelog Please remove this and update the metadata to ensure we grab the correct version.